src/containers/pages/InterventionPlan/UpdatePlan/index.tsx

```diff
@@ -5,6 +5,9 @@ import { connect } from 'react-redux';
 import { RouteComponentProps } from 'react-router';
 import { Col, Row } from 'reactstrap';
 import { Store } from 'redux';
+import ConnectedJurisdictionIdToName, {
+  JurisdictionIdsToNames,
```

Comment:
I think that this should be `JurisdictionIdToName`.  If that is the case, why is a test not failing?  What test do we need to add?

Reply:
Actually I think that this is an unused import.

Reply:
There isn't yet an integration test for JurisdictionIdToName in updatePlan. will add one.